src/components/Form/fields.jsx

```diff
@@ -1,27 +1,22 @@
 import React from 'react'
-import Input from 'cozy-ui/transpiled/react/Input'
-import Textarea from 'cozy-ui/transpiled/react/Textarea'
+import TextField from 'cozy-ui/transpiled/react/MuiCozyTheme/TextField'
 
-function adaptField(WrappedComponent) {
-  class AdaptField extends React.Component {
-    render() {
-      const {
-        input,
-        meta: { valid },
-        ...rest
-      } = this.props
-      return <WrappedComponent {...input} {...rest} data-valid={valid} />
```

Comment:
Did you find out why the `data-valid` was there?

Reply:
Non pas du tout !

Reply:
We checked together, as far as we can tell it had no practical utility.